utils/resultwriter.go

```diff
@@ -48,10 +49,30 @@ type ResultsWriter struct {
 	messages []string
 }
 
+type Vulnerability struct {
+	BomRef string `json:"bom-ref" xml:"bom-ref,attr"`
+	ID     string `json:"id" xml:"id"`
+}
+
+type XMLVulnerability struct {
+	Vulnerability []Vulnerability `xml:"vulnerability"`
+}
+
+type Vulnerabilities struct {
+	Vulnerabilities XMLVulnerability `xml:"vulnerabilities"`
+}
```

Comment:
You have 2 different def for this one in here and one in the `enrich_test`. if this the output format, move it to its own file under `formats` dir and use it from there

Reply:
ok